src/crl.rs

```diff
@@ -20,28 +20,28 @@ use crate::{der, signed_data, Error, Time};
 ///
 /// [^1]: <https://www.rfc-editor.org/rfc/rfc5280#section-5>
 pub struct BorrowedCertRevocationList<'a> {
-    /// A `SignedData` structure that can be passed to `verify_signed_data`.
-    pub(crate) signed_data: signed_data::SignedData<'a>,
-
-    /// Identifies the entity that has signed and issued this
-    /// CRL.
-    pub(crate) issuer: untrusted::Input<'a>,
+    /// A monotonically increasing sequence number for a given CRL scope and CRL issuer.
+    pub crl_number: Option<&'a [u8]>,
 
     /// Indicates the issue date of this CRL.
     pub this_update: Time,
 
     /// Indicates the date by which the next CRL will be issued.
     pub next_update: Time,
 
+    /// A `SignedData` structure that can be passed to `verify_signed_data`.
+    pub(crate) signed_data: signed_data::SignedData<'a>,
```

Comment:
Yeah, I think keeping them private is the right default: making them public in a semver-compatible release is trivial, whereas changing anything about a public field is much less so.

Reply:
I've made the fields private. I had to add some dead code allows as a result. If we want to avoid those I could rip out the fields + associated parsing entirely. LMK what you prefer.